html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <label for="email">Email</label>
+      <div>
+        <input type="email" name="email" id="email" required>
+      </div>
+      <label for="coding-cool">Why is coding cool?</label>
+      <div>
+        <select name="reason" id="coding-cool">
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <label for="html">
+          <input type="radio" name="language" value="html"/> HTML
```

Comment:
If you are using the `for` attribute on the `label` tag is there a matching attribute that needs to be on the `input` for it to work properly?

The user should be able to click on the text of the label to select the input as well.

Reply:
There has to be an id attribute matching the for attribute. Can the value and id attributes have the same values?

Reply:
good question - yes it's okay for those to be the same